python/cugraph/cugraph/dask/structure/mg_property_graph.py

```diff
@@ -437,18 +437,20 @@ def get_vertex_data(self, vertex_ids=None, types=None, columns=None):
         vertex_ids, columns, and/or types, or all vertex IDs if not specified.
         """
         if self.__vertex_prop_dataframe is not None:
+            df = self.__vertex_prop_dataframe
             if vertex_ids is not None:
-                df_mask = (
-                    self.__vertex_prop_dataframe[self.vertex_col_name]
-                    .isin(vertex_ids)
-                )
-                df = self.__vertex_prop_dataframe.loc[df_mask]
-            else:
-                df = self.__vertex_prop_dataframe
+                if isinstance(vertex_ids, int):
+                    df_mask = df[self.vertex_col_name] == vertex_ids
+                else:
+                    df_mask = df[self.vertex_col_name].isin(vertex_ids)
+                df = df.loc[df_mask]
 
             if types is not None:
                 # FIXME: coerce types to a list-like if not?
```

Comment:
is this FIXME still relevant, since it seems like we decided we aren't coercing to a list?  Same question for all the other FIXMEs like this.

Reply:
well, we check if it's a string, but we don't yet check if it's a list.  We defer to `isin`.  so if we don't have a clear answer to this FIXME, then it _may_ still be relevant.